tests/unit_tests/test_wave_06.py

```diff
@@ -228,16 +223,16 @@ def test_swap_best_by_category_no_match_is_false():
         their_priority="Clothing"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
 
-@pytest.mark.skip
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert result is False
+    assert item_a and item_b and item_c in tai.inventory 
+    assert item_d and item_e and item_f in jesse.inventory
```

Comment:
These lines don't do what you want them to do!

Consider this Python code to demonstrate it:

```python
jesse = ["d", "e", "f"]
print("a" in jesse) # Prints False
print("d" in jesse) # Prints True
print("a" and "d" in jesse) # Prints True (Should print False! "a" is NOT in `jesse`!)
print("a" in jesse and "d" in jesse) # Prints False as expected
```

When you say `assert item_d and item_e and item_f in jesse.inventory`, you're _not_ saying "check that `item_d`, `item_e`, and `item_f` are in `jesse.inventory`.

Python reads this as "Is the expression `item_d` truthy? _and_ is the expression `item_e` truthy? _and_ is the expression `item_f in jesse.inventory` truthy?" Therefore, to fix this, you'd need:

```python
assert item_d in jesse.inventory and item_e in jesse.inventory and item_f in jesse.inventory
```